Cargo.toml

```diff
@@ -65,6 +65,8 @@ dirs-next = { version = "2.0.0", optional = true }
 grep-cli = { version = "0.1.6", optional = true }
 regex = { version = "1.0", optional = true }
 walkdir = { version = "2.0", optional = true }
+time = { version = "0.3.5", features = ["formatting"] }
```

Comment:
I just noticed this breaks our minimum rust version support, as the minimum supported version of this library is 1.51. If necessary, I can switch to `chrono` or some other library for formatting the timestamp.

Reply:
Hi and thanks for making a contribution! I don't have time to review at the moment, but me or someone else can probably look ath these in max a few weeks

however, I can already now say that bat will soon bump to 1.51 as well, because syntect already has, and bat will soon depend on the soon to be made syntect 4.7.1 release

Reply:
In fact, feel free to create a PR that bumps MSRV to 1.51, I would set Approved on such a PR

Reply:
Hello! Sounds good, will create a MR for that as well. No rush for reviewing the MR either, but thanks for letting me know in advance.

Reply:
https://github.com/sharkdp/bat/pull/1994